data/serializer/src/property-test/java/tech/pegasys/teku/provider/JsonProviderPropertyTest.java

```diff
@@ -420,40 +417,4 @@ void roundTripKZGCommitment(@ForAll final int seed) throws JsonProcessingExcepti
     final KZGCommitment deserialized = jsonProvider.jsonToObject(serialized, KZGCommitment.class);
     assertThat(deserialized).isEqualTo(original);
   }
-
-  @Property
-  void roundTripBlobsSidecar(@ForAll final int seed) throws JsonProcessingException {
-    final SpecMilestone specMilestone = SpecMilestone.DENEB;
-    final Spec spec = TestSpecFactory.create(specMilestone, Eth2Network.MINIMAL);
-    final DataStructureUtil dataStructureUtil = new DataStructureUtil(seed, spec);
-    final BlobsSidecar original = new BlobsSidecar(dataStructureUtil.randomBlobsSidecar());
-    final String serialized = jsonProvider.objectToJSON(original);
-    final BlobsSidecar deserialized = jsonProvider.jsonToObject(serialized, BlobsSidecar.class);
-    assertThat(deserialized).usingRecursiveComparison().isEqualTo(original);
-  }
-
-  @Property
-  void roundTripSignedBeaconBlockAndBlobsSidecar(@ForAll final int seed)
-      throws JsonProcessingException {
-    final SpecMilestone specMilestone = SpecMilestone.DENEB;
-    final Spec spec = TestSpecFactory.create(specMilestone, Eth2Network.MINIMAL);
-    final DataStructureUtil dataStructureUtil = new DataStructureUtil(seed, spec);
-    final SignedBeaconBlockAndBlobsSidecar original =
-        new SignedBeaconBlockAndBlobsSidecar(
-            dataStructureUtil.randomSignedBeaconBlockAndBlobsSidecar());
-    final String serialized = jsonProvider.objectToJSON(original);
-    final SignedBeaconBlockAndBlobsSidecar deserialized =
-        jsonProvider.jsonToObject(serialized, SignedBeaconBlockAndBlobsSidecar.class);
-    final SignedBeaconBlockAndBlobsSidecarSchema signedBeaconBlockAndBlobsSidecarSchema =
-        spec.getGenesisSchemaDefinitions()
-            .toVersionDeneb()
-            .orElseThrow()
-            .getSignedBeaconBlockAndBlobsSidecarSchema();
-    assertThat(
-            deserialized.asInternalSignedBeaconBlockAndBlobsSidecar(
-                signedBeaconBlockAndBlobsSidecarSchema, spec))
-        .isEqualTo(
-            original.asInternalSignedBeaconBlockAndBlobsSidecar(
-                signedBeaconBlockAndBlobsSidecarSchema, spec));
-  }
 }
```

Comment:
We could add in nice to have in decoupling issue to make it for BlobSidecar and other new structures probably